lib/gc/Transforms/GPU/LinalgToXeGPU.cpp

```diff
@@ -1057,7 +1076,7 @@ static LogicalResult createDPASKernel(linalg::LinalgOp linalgOp,
 
   // Load A sub-tiles.
   SmallVector<Value> loadVecA =
-      loadNdDescTiles(rewriter, loc, tilesA, readCacheHint, vnniConfA);
```

Comment:
`vnniConfA` can't be used during loading since `vnniAxis=1` is now longer supported. However we still need this config to compute proper tiles for `xegpu.dpas` later in the code.